src/c_kzg_4844.c

```diff
@@ -698,90 +698,52 @@ static C_KZG_RET blob_to_polynomial(Polynomial *p, const Blob *blob) {
     return C_KZG_OK;
 }
 
-/* Forward function definition */
-static void compute_powers(fr_t *out, fr_t *x, uint64_t n);
+/* Input size to the Fiat-Shamir challenge computation */
+#define CHALLENGE_INPUT_SIZE 32 + BYTES_PER_BLOB + 48
```

Comment:
Would `static const int` be better here? or even `enum`?

Reply:
Hmm. I think it's standard (and reasonable) practice to have public constants as `#define`s. We do the same thing for `BYTES_PER_COMMITMENT` etc.

I don't see a clear benefit to making it `static const int`, and I'm not sure how we would use an `enum`.

Reply:
Some thoughts on the benefits can be found here https://stackoverflow.com/questions/1674032/static-const-vs-define-vs-enum. In this case I was thinking of it primarily because we're doing arithmetic (`+`) unlike `BYTES_PER_COMMITMENT` which is a bare integer literal. (`BYTES_PER_BLOB` however would face the same argument.) We count on the compiler to constant fold this arithmetic when using macros.

Reply:
No strong opinion here. I will defer to @jtraglia .

Reply:
yeah to be clear I don't really have a strong opinion either.

Reply:
No strong opinion either, but I'm leaning towards a static const. In my opinion it's safer. If you decide to stick with a macro, you should wrap it in parentheses in-case it's every used in a multiplication/division operation. This was a problem with `BYTES_PER_BLOB` at one point in time.

Reply:
ACK! Fixed in https://github.com/ethereum/c-kzg-4844/pull/123/commits/53534347ec7d0d16f8be84aa0285916bcf7ac798 .